msteams-platform/messaging-extensions/create-api-message-extension.md

```diff
@@ -136,10 +136,16 @@ To build am API-based message extension using Teams Toolkit for Visual Studio Co
 
    # [New API](#tab/new-api)
```

Comment:
There is a typo at the beginning of the doc:

![Uploading image.png…]()

Reply:
updated.